developer/doc/ReleaseNotes/OpenStudio_Release_Notes_3_8_0_TBD.md

```diff
@@ -77,7 +77,21 @@ You can also refer to the [OpenStudio SDK Python Binding Version Compatibility M
 
 For a list of deprecated and removed methods, please refer to [deprecated_methods.csv](../../ruby/deprecated_methods.csv)
 
-* [#TDB]() - Update to EnergyPlus 24.1.0
+* [#5094](https://github.com/NREL/OpenStudio/pull/5094) - Update to EnergyPlus 24.1.0
+
+* [#5137](https://github.com/NREL/OpenStudio/pull/5137) - Update from Ruby 2.7.2 to Ruby 3.2.2. This required a significant effort in other dependencies including Conan, and significant work in the continuous integration (CI) system.
+
+* [#TBD]() Performance Improvements to OpenStudio logging. 
```

Comment:
@kbenne I put place holder here for logging performance improvements, but plan to add a lot more details. Free fee to take first stab at this if you want and have time.